daffodil-core/src/main/scala/org/apache/daffodil/runtime1/SchemaSetRuntime1Mixin.scala

```diff
@@ -73,6 +73,10 @@ trait SchemaSetRuntime1Mixin { self : SchemaSet =>
 
   def onPath(xpath: String): DFDL.DataProcessor = {
       Assert.usage(!isError)
+      if (false) {
+        println("test of code cov report")
+      }
```

Comment:
Seems like it's working to me.

Reply:
Yep, I guess I was expected an overall PR comment to be made (It used to do this), but maybe that's been disabled in place of inline comments, which I guess is fine.

One thought, PR #427 that you opened now fails because of the codecov bot being enabled. For some reason that PR decreases coverage on the overall project. Looking at the report, it's not clear what coverage is actually missing. I'm wondering if maybe it just comes down to the fact that the PR deleted a file that maybe had full coverage, and so be removing that causes less overall coverage? I'd be surprised if there was any real change in code paths between 2.11 and 2.12. But maybe?

I'm not sure if there's a good fix. We could disable ``codevoc/project`` check and only require that PR's have coverage of what they change, but it would be nice to be alerted if a PR removes tests or something that decreases overall coverage.

Reply:
The total coverage on the project in your PR's codecov report was only `80.21% (+0.01%) compared to d3b6350` and the total coverage on the project in my PR's codecov report went down to `78.29% (+-1.93%) compared to 82028c1`.

My PR indeed deletes one file (the file that was in the Scala 2.11-only src directory).  That deletion must have pushed the project's total coverage down below 80%.  

We could add some test cases to my PR to push the total coverage above 80% again.  It might even take only 1-2 test cases if we identify a class that had no test coverage to start with.  You're more familiar with the codecov UI and the codebase than I am.  Will you please identify a class that we could add a test to?

Reply:
If adding more tests becomes too onerous to do, then we also could reduce the 80% minimum coverage limit to 75%.  That limit's probably a parameter we could configure somewhere.  But let's hope adding more tests is as easy to do.

Reply:
I suggest create a JIRA ticket "Increase code test coverage" and move on for now.  Time spent on improving coverage should be done with an intention to add coverage where it is most needed, not just to get back to some artificial figure of merit that honestly, was never a target. It's just where we landed when we first turned on coverage testing.

Reply:
This is causing all the PR's to appear as if they are failing. It seems even the slightest change can cause a decrease in project coverage. Since we aren't actively trying to maintain some certain percentage coverage, it doesn't really make sense to have this fail PRs. I'll add a config option to disable the codecov.project. I think it's still worth keeping codecov/patch as we should probably strive to have coverage for all new code.